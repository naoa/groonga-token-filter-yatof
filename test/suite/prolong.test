register token_filters/yatof

tokenize TokenDelimit \
  "フリーザー カー ブザー ﾃﾞｰﾀｰ がーたー" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterProlong
