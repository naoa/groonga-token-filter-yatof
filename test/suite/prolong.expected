register token_filters/yatof
[[0,0.0,0.0],true]
tokenize TokenDelimit   "フリーザー カー ブザー ﾃﾞｰﾀｰ がーたー"   --normalizer NormalizerAuto   --token_filters TokenFilterProlong
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "フリーザ",
      "position": 0
    },
    {
      "value": "カー",
      "position": 1
    },
    {
      "value": "ブザー",
      "position": 2
    },
    {
      "value": "データ",
      "position": 3
    },
    {
      "value": "がーたー",
      "position": 4
    }
  ]
]
